### DISCUSSION TITLE
linux application emulator

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#252] [Feature Requests]
is there something that could work on this like boxedwine, but just for Linux? like, say you run a .deb file, and it runs a program to auto-install it, and then run it in a window.

**Comment by DustinBrett** [comment]
Would be cool and v86 can do a bit of this, but nothing like BoxedWine afaik. Nothing that works.

https://github.com/copy/v86

**Comment by DustinBrett** [comment]
There is also box86js, but it doesn't seem to work well.

https://github.com/raoofha/box86js
